script/licenses.py

```diff
@@ -0,0 +1,275 @@
+    "GNU Library or Lesser General Public License (LGPL)",
+    "Historical Permission Notice and Disclaimer (HPND)",
+    "IBM Public License",
+    "ISC License (ISCL)",
+    "Intel Open Source License",
+    "Jabber Open Source License",
+    "MIT License",
+    "MIT No Attribution License (MIT-0)",
+    "MITRE Collaborative Virtual Workspace License (CVW)",
+    "MirOS License (MirOS)",
+    "Motosoto License",
+    "Mozilla Public License 1.0 (MPL)",
+    "Mozilla Public License 1.1 (MPL 1.1)",
+    "Mozilla Public License 2.0 (MPL 2.0)",
+    "Mulan Permissive Software License v2 (MulanPSL-2.0)",
+    "NASA Open Source Agreement v1.3 (NASA-1.3)",
+    "Nethack General Public License",
+    "Nokia Open Source License",
+    "Open Group Test Suite License",
+    "Open Software License 3.0 (OSL-3.0)",
+    "PostgreSQL License",
+    "Python License (CNRI Python License)",
+    "Python Software Foundation License",
+    "Qt Public License (QPL)",
+    "Ricoh Source Code Public License",
+    "SIL Open Font License 1.1 (OFL-1.1)",
+    "Sleepycat License",
+    "Sun Industry Standards Source License (SISSL)",
+    "Sun Public License",
+    "The Unlicense (Unlicense)",
+    "Universal Permissive License (UPL)",
+    "University of Illinois/NCSA Open Source License",
+    "Vovida Software License 1.0",
+    "W3C License",
+    "X.Net License",
+    "Zero-Clause BSD (0BSD)",
+    "Zope Public License",
+    "zlib/libpng License",
+    "Apache License",
+    "MIT",
+    "apache-2.0",
+    "GPL-3.0",
+    "GPLv3+",
+    "MPL2",
+    "MPL-2.0",
+    "Apache 2",
+    "LGPL v3",
+    "BSD",
+    "GNU-3.0",
+    "GPLv3",
+    "Eclipse Public License v2.0",
+    "ISC",
+    "GPL-2.0-only",
+    "mit",
+    "GNU General Public License v3",
+    "Unlicense",
+    "Apache-2",
+    "GPLv2",
+}
+
+EXCEPTIONS = {
+    "PyMicroBot",  # https://github.com/spycle/pyMicroBot/pull/3
+    "PySwitchmate",  # https://github.com/Danielhiversen/pySwitchmate/pull/16
+    "PyXiaomiGateway",  # https://github.com/Danielhiversen/PyXiaomiGateway/pull/201
+    "aiocomelit",  # https://github.com/chemelli74/aiocomelit/pull/138
+    "aioecowitt",  # https://github.com/home-assistant-libs/aioecowitt/pull/180
+    "aiohappyeyeballs",  # PSF-2.0 license
+    "aiohttp-fast-url-dispatcher",  # https://github.com/bdraco/aiohttp-fast-url-dispatcher/pull/10
+    "aioopenexchangerates",  # https://github.com/MartinHjelmare/aioopenexchangerates/pull/94
+    "aiooui",  # https://github.com/Bluetooth-Devices/aiooui/pull/8
+    "aioruuvigateway",  # https://github.com/akx/aioruuvigateway/pull/6
+    "aiovodafone",  # https://github.com/chemelli74/aiovodafone/pull/131
+    "airthings-ble",  # https://github.com/Airthings/airthings-ble/pull/42
+    "amberelectric",  # https://github.com/madpilot/amberelectric.py/pull/11
+    "apple_weatherkit",  # https://github.com/tjhorner/python-weatherkit/pull/3
+    "asyncio",  # Python core
```

Comment:
FYI `asyncio` in that case is a separate Python package not the python core module.
https://pypi.org/project/asyncio/